site/app/libraries/database/DatabaseQueriesPostgresql.php

```diff
@@ -303,12 +305,12 @@ public function getGradeables($g_ids = null, $user_ids = null, $section_key="reg
             $query .= "
 WHERE ".implode(" AND ", $where);
         }
-
         if ($user_ids !== null) {
-            $query .= "
-ORDER BY u.{$section_key}, u.user_id";
+          $query .= "
+ORDER BY u.{$section_key}, {$sort_key}";
```

Comment:
Same comment as above that we should not just be sorting by first name or last name (though just by id is fine as that's unique).